torch/onnx/symbolic_opset23.py

```diff
@@ -0,0 +1,93 @@
+# mypy: allow-untyped-defs
+"""This file exports ONNX ops for opset 23.
+
+Note [ONNX Operators that are added/updated in opset 23]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+https://github.com/onnx/onnx/blob/main/docs/Changelog.md#version-23-of-the-default-onnx-operator-set
+New operators:
+    Attention
+    RMSNormalization
+    Reshape
+    RotaryEmbedding
+"""
+
+import functools
+from torch import _C
+from torch.onnx import symbolic_helper
+from torch.onnx._internal import jit_utils, registration
+
+__all__ = ["attention", "rms_normalization", "reshape", "rotary_embedding"]
+
+_onnx_symbolic = functools.partial(registration.onnx_symbolic, opset=23)
+
+
+@_onnx_symbolic("aten::attention")
+@symbolic_helper.parse_args("v", "v", "v", "v", "v", "v", "i", "i", "i", "f", "f", "i")
+def attention(
```

Comment:
These ops do not seem to exist as ATen operators. You may start with implementing scaled_dot_product_attention which has the following signature:

https://github.com/pytorch/pytorch/blob/3aa84775e7e63799cda90f406f44c25b688ecce0/aten/src/ATen/native/native_functions.yaml#L14880